ga4gh/cli.py

```diff
@@ -181,6 +127,109 @@ def createGAListReferenceBasesRequest(self):
         return request
 
 
+def getWorkarounds(args):
+    if args.workarounds is None:
+        return set()
+    else:
+        return set(args.workarounds.split(','))
+
+
+##############################################################################
+# ga2vcf
+##############################################################################
+
+
+def addOutputFileArgument(parser):
+    parser.add_argument(
+        "--outputFile", default=None,
+        help="the file to write the output to")
+
+
+def ga2vcf_main(parser=None):
+    # parse args
+    if parser is None:
+        parser = argparse.ArgumentParser(
+            description="GA4GH VCF conversion tool")
+    addClientGlobalOptions(parser)
+    addOutputFileArgument(parser)
+    subparsers = parser.add_subparsers(title='subcommands',)
+    variantsSearchParser = addVariantsSearchParser(subparsers)
+    # TODO kinda ugly that we need to overload the parser with arguments
+    # to populate the searchVariantSetsRequest
+    addDatasetIdsArgument(variantsSearchParser)
+    args = parser.parse_args()
+    if "runner" not in args:
+        parser.print_help()
+    else:
+        ga2vcf_run(args)
+
+
+def ga2vcf_run(args):
+    # instantiate params
+    searchVariantsRequest = RequestFactory(
+        args).createGASearchVariantsRequest()
+    searchVariantSetsRequest = RequestFactory(
+        args).createGASearchVariantSetsRequest()
+    if args.outputFile is None:
+        outputStream = sys.stdout
+    else:
+        outputStream = file(args.outputFile, 'w')
+    workarounds = getWorkarounds(args)
+    httpClient = client.HttpClient(
+        args.baseUrl, args.verbose, workarounds, args.key)
+
+    # do conversion
+    vcfConverter = converters.VcfConverter(
+        httpClient, outputStream,
+        searchVariantSetsRequest, searchVariantsRequest)
+    vcfConverter.convert()
+
+    # cleanup
+    if args.outputFile is not None:
+        outputStream.close()
+
+
+##############################################################################
+# ga2sam
+##############################################################################
+
+
+def ga2sam_main(parser=None):
+    # parse args
+    if parser is None:
+        parser = argparse.ArgumentParser(
+            description="GA4GH SAM conversion tool")
+    addClientGlobalOptions(parser)
+    addReadsSearchParserArguments(parser)
+    addBinaryOutputArgument(parser)
+    addOutputFileArgument(parser)
```

Comment:
Not sure if it's defined here, but add a short option for --outputFile also. We should support both '-o file' and '--outputfile file'